### PR TITLE
Mentioned that older versions may need /v2/

### PR DESCRIPTION
**Pull request by cromefire**
See #411

### CODE REVIEW

File: users/stdiscosrv.rst [hunk 1]
```diff
@@ -68,6 +68,10 @@ list, e.g. ``https://disco.example.com:8443/``. Note that stdiscosrv uses port
 8443 by default. For stdiscosrv to be available over the internet with a dynamic
 IP address, you will need a dynamic DNS service.
 
+.. warning::
+   Versions prior to ``0.14.44`` may need ``/v2/`` appended to the discovery
+   server address, e.g. ``https://disco.example.com:8443/v2/``.
```
Comment (imsodin) [inline users/stdiscosrv.rst]: I'd use `.. deprecated:: v0.14.44` and remove the may, something like 
> Prior versions needed ``v2/`` appended to the discovery  server address, e.g. ``https://disco.example.com:8443/v2/``."

Comment (cromefire) [reply users/stdiscosrv.rst]: I think depending on you version (for example if it's even older) it may no be needed.

**Comment by AudriusButkevicius** [comment]
Lets get rid of the may, discovery server versions prior to tuat probably don't exist.

**Comment by cromefire** [comment]
There you go

File: users/stdiscosrv.rst [hunk 2]
```diff
@@ -68,6 +68,10 @@ list, e.g. ``https://disco.example.com:8443/``. Note that stdiscosrv uses port
 8443 by default. For stdiscosrv to be available over the internet with a dynamic
 IP address, you will need a dynamic DNS service.
 
+.. deprecated:: v0.14.44
+   Prior version need ``/v2/`` appended to the discovery
```
Comment (imsodin) [inline users/stdiscosrv.rst]: I feel a bit silly about the amount of review going into such a little change (thanks already for your patience), but could do version->versions.

Comment (cromefire) [reply users/stdiscosrv.rst]: It would have saved me 6 hours of time so it's relatively important I think, but it okay, I know that

**Comment by cromefire** [comment]
All fixed

**Pull request merged**